src/themes/default/components/tab.js

```diff
@@ -0,0 +1,82 @@
+const baseStyle = {
+  fontFamily: 'fontFamilies.ui'
+}
+
+const appearances = {
+  primary: {
+    color: 'colors.muted',
+    paddingTop: '6px',
+    paddingBottom: '16px',
+    paddingLeft: '2px',
+    paddingRight: '2px',
+    position: 'relative',
+
+    ':not(:last-child)': {
+      marginRight: '20px'
+    },
+    
+    ':before': {
```

Comment:
Can we just bridge support for this by adding pseudo placeholders for `_before` and `_after`?